GitHub-Test-4/nlp2.py

```diff
@@ -0,0 +1,36 @@
+import os
+import spacy
+#nlp = spacy.cli.download("en_core_web_md")
+nlp = spacy.load('en_core_web_md')
+workingDir = os.getcwd()
+print("current working directory:" + workingDir)
+insideDir = os.listdir(workingDir)
+print("inside this directory are the following files AND directories: " + str(insideDir))
+CollPath = os.path.join(workingDir, 'aitexts')
+#print(CollPath)
+#aitext has three documents. Two of them have the same prompt.
+def readTextFiles(filepath):

```

Comment:
@andrewjack7887 This is an example of a line comment, here used to highlight the first line of @sammoniot 's `readTextFiles()` function.